conda_forge_tick/auto_tick.py

```diff
@@ -449,22 +451,29 @@ def add_rebuild_migration_yaml(
         package_names,
         excluded_feedstocks,
         include_noarch=config.get("include_noarch", False),
+        include_by_build_only=config.get("include_by_build_only", False),
     )
 
     # Note at this point the graph is made of all packages that have a
     # dependency on the pinned package via Host, run, or test.
     # Some packages don't have a host section so we use their
     # build section in its place.
 
-    package_names = {
-        p if p in gx.nodes else output_to_feedstock[p] for p in package_names
-    } - excluded_feedstocks
+    if "override_cbc_keys" not in config:
+        # this will fail if we've not used keys in the cbc that are actual
+        # packages in the graph - thus we skip it in that case
+        package_names = {
+            p if p in gx.nodes else output_to_feedstock[p] for p in package_names
```

Comment:
```suggestion
            output_to_feedstock.get(p, p) for p in package_names
```
This should handle things like proj/proj.4

Reply:
I am also worried that putting a compiler stub through this block of logic will error, but I can test.

Reply:
if the compiler stubs error it will error either way, I don't think they are in the outputs table